frontend/tests/utils/authHelpers.ts

```diff
@@ -37,6 +38,23 @@
   return userId;
 }
 
+export async function addUserToOrg(api: APIRequestContext, userId: string, orgId: string, role: OrgRole): Promise<UUID> {
```

Comment:
this function probably isn't returning a `UUID` as `executeGql` just returns the raw json from the request which might be something like `{data:{changeOrgMemberRole:{organization:{id: 'value"}}}}` but it would only be returning the org id which was just passed in so I don't think we even care about the result.

Reply:
Changed to `Promise<unknown>` in commit e342c9626.